docs/10_utilities/fill-queue/fill-queue-how-to.md

```diff
@@ -0,0 +1,100 @@
+---
+content_title: How-To Setup And Use Fill-Queue
+link_text: How To Setup and Use Fill-Queue
+---
+
+## Summary
+
+This how-to demonstrates the set up and use of [Fill-Queue application](index) running locally.
+
+## Prerequisites
+
+Before you begin, make sure you have the following information or have completed the following referenced procedures:
+
+1. `nodeos` runs locally.
+2. `nodeos` has the state history plugin enabled. \
+For instructions on how to start `nodeos` with history plugin enabled consult
+[the following documentation](https://developers.eos.io/manuals/eos/v2.1/nodeos/plugins/state_history_plugin/index/).
+3. RabbitMQ runs locally.
+4. OpenSSL is installed locally.
```

Comment:
install package `locally` sounds confusing. Normally everything is installed locally unless you ssh to some host.

Reply:
the whole how to is about running locally, it is explicitly stated at the very beginning.
I don't believe it is hurting to say it here again, it emphasising the local context for all the parts involved: rabitmq, nodeos, fill-queue, openssl.

what I am concerned though in your observation is the confusing part... can you provide an alternative or you just want it out completely for all 1, 3 and 4 bullets? they all mention "locally" context.

Reply:
nodeos and rabbitmq can run remotely, word locally is fine here.
But remote installation is something very specific. So locally installed sounds a bit weird. I would write `OpenSSL is installed on the same machine nodeos is running`. Or simply mention at the beginning:
`Do this on the same machine:`
Last one sounds to me like best way to describe what you want to do.
And also any installation should preceded ant execution. So openssl step should be 1st i think.